brunch-config.js

```diff
@@ -31,8 +31,7 @@ exports.config = {
   plugins: {
     babel: {
       // Do not use ES6 compiler in vendor code
-      ignore: [/^(web\/static\/vendor)/],
-      loose: "all"
```

Comment:
> What are we gaining from upgrading to babel 6? If there is no clear benefit maybe we should wait with the update until there are benefits.

I am generally in agreement here, but staying behind the curve on anything in the node ecosystem is a problematic. Is it right to say that we could upgrade babel as proposed, keep xdomain, for IE9/10, and that folks that still need IE8 can include that list of plugin options? If so it seem like everyone's happy? If you're stuck supporting ie8, copying 15 lines of config is probably the easiest job you'll have that day :)